assets/src/blocks/Happypoint/HappypointFrontend.js

```diff
@@ -42,19 +44,24 @@ export const HappypointFrontend = ({
       </picture>
       <div className="container">
         <div className="row justify-content-md-center">
-          {mailing_list_iframe && url && (
-            <div className="col-md-10 happy-point" id="happy-point" data-src={url}>
-              <iframe
-                src={url}
-                cellSpacing={0}
-                allowtransparency="true"
-                frameBorder={0}
-                scrolling="no"
-                width="100%"
-                loading="lazy"
-              />
-            </div>
-          )}
+          {(use_embed_code && embed_code) ?
+            <div dangerouslySetInnerHTML={{ __html: embed_code }} /> :
```

Comment:
That works fine for simple html code, but it doesn't seem to run any `<script>` tags that are part of the code.